src/NovelRT/Windowing/WindowingService.cpp

```diff
@@ -36,6 +36,11 @@ namespace NovelRT::Windowing {
       checkForOptimus(OptimusLibraryName);
 #endif
 
+    //Set Framebuffer Transparency
+   if (transparencyEnabled)
+   {
+     glfwWindowHint(GLFW_TRANSPARENT_FRAMEBUFFER, GLFW_TRUE);
+   }
```

Comment:
Sets the framebuffer to be transparent. Must be done on the Window level before the OpenGL context is initialized.

Reply:
Does this account for all possible attempts to create the window?

Also your bracing is off.

Reply:
We have different types of bracing throughout our code - can you remind me which style we're to use going forward?
And yes, since this is called before OpenGL context initialization, it is cached for all attempts.

Reply:
For the time being were still using K&R. We agreed to switch to Allman once we enforced the .NET style everywhere via `clang-format`, but that hasn't been done yet, so its best to just stick to the old style until we go over.

Reply:
Fixed to K&R style. And to answer the previous question (as a reminder from Discord), this accounts for all window creation attempts :D